fiducial_slam/src/map.cpp

```diff
@@ -69,13 +70,13 @@ static double updateVarianceDavid(const tf2::Vector3 &newMean,
     double d1 = (mean1 - newMean).length2();
     double d2 = (mean2 - newMean).length2();
 
-    double newVar = sqrt(2.0*M_PI) * var1 * var2 *
+    double newVar = systematic_error + sqrt(2.0*M_PI) * var1 * var2 *
          exp(((d1 / (2.0*var1)) + d2 / (2.0*var2)));
 
     if (newVar > 100)
         newVar = 100;
-    if (newVar < 10e-4)
```

Comment:
I am OK with leaving the clamps in for now, but I think we should take them out right after testing or we should convert them in to a warning rather than an absolute clamp. Right now these clamps have the ability to create unexpected behavior. What you are saying with these clamps is that the uncertainty can never be less than 100 microns and never be more than 100 meters. 

I don't know how we can be certain of that 100% of the time so we probably shouldn't imply that in the code. Much better to say - hey your error numbers look really weird, perhaps you better do something about that - in otherwords raise it to the level of human attention.